codecov_auth/tests/test_migrations.py

```diff
@@ -1,108 +0,0 @@
-from datetime import datetime
```

Comment:
Remove this test for previous migration since these columns are deleted in this change anyways.